style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  background-color: #fff;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.profile {
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 45px;
+  font-family: "poppins", sans-serif;
+  line-height: 50px;
+}
```

Comment:
- [ ] Please use the same styling for the `<h1>` tag as the given design.

_Look up the comment about the given design template._